File: github-api-c#/GitHubTests/GitHub/UI/IssuesPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace GitHubTests.GitHub.UI
{
    class IssuesPage
    {
        private string url = "https://github.com/dtopuzov/test/issues";
        private IWebDriver _driver;

        [FindsBy(How = How.XPath, Using = "//*[@id=\"js-issues-toolbar\"]/div[1]/div[1]/a[2]")]
        private IWebElement closedIssues { get; set; }

        [FindsBy(How = How.XPath, Using = "//*[@id=\"js-issues-toolbar\"]/div[1]/div[1]/a[1]")]
        private IWebElement openIssues { get; set; }

        public IssuesPage(IWebDriver driver)
        {
            this._driver = driver;
            PageFactory.InitElements(_driver, this);
        }

        public void NavigateTo()
        {
            this._driver.Navigate().GoToUrl(url);
        }

        public int GetClosedIssuesCount()
        {
            return int.Parse(this.closedIssues.Text.Split(' ')[0]);
        }

        public int GetOpenIssuesCount()
        {
            return int.Parse(this.openIssues.Text.Split(' ')[0]);
        }
    }
}

File: reset-assured/src/main/java/GitHub/UI/IssuesPage.java
package GitHub.UI;

import Base.UI.BasePage;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.CacheLookup;
import org.openqa.selenium.support.FindBy;

public class IssuesPage extends BasePage {

    @FindBy(xpath = "//*[@id=\"js-issues-toolbar\"]/div[1]/div[1]/a[2]")
    @CacheLookup
    private WebElement closedIssues;

    @FindBy(xpath = "//*[@id=\"js-issues-toolbar\"]/div[1]/div[1]/a[1]")
    @CacheLookup
    private WebElement openIssues;

    public IssuesPage(WebDriver driver) {
        super(driver);
        driver.navigate().to("https://github.com/dtopuzov/test/issues");
    }

    public int getClosedIssuesCount() {
        return Integer.parseInt(closedIssues.getText().split(" ")[0]);
    }

    public int getOpenIssuesCount() {
        return Integer.parseInt(openIssues.getText().split(" ")[0]);
    }
}